rust/core-lib/src/editor.rs

```diff
@@ -295,6 +295,10 @@ impl Editor {
         Some((delta, last_text, keep_selections))
     }
 
+    pub(crate) fn delta_between(&self, target_rev_id: u64) -> Delta<RopeInfo> {
```

Comment:
I'd just call this `delta_rev_head` as well, just because it makes it easier to talk about the code if this functionality has a single name. I'd also make the `u64` argument be a `RevToken`, which is just a typedef but is consistent with other use.